### PR TITLE
135 in app signup

### LINKED ISSUE #135: A user can create an account from within the app
## TODO

- [ ] Generate design asset to reflect UI

Rough mockup:

image

spacing / padding probably isn't right...this is rough. But I think this could be nice.

-----

## User Story

A user wants to easily sign up for a new account from within the app.

#### Problem Details _(Optional)_

* Screenshots, GIFs, or a video
* Any additional information (version, device, etc.)

#### Acceptance Criteria _(Optional)_

* [ ] A user can create an account from within the app
* [ ] Turn the login screen to a two-tab view (similar to NS/Apollos) with Login on one tag, Signup on the other

### PR DESCRIPTION
**Pull request by conrad-vanl**
Fixes #135 

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings (in app AND in storybook)
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [x] Add a line to the changelog documenting the high-level change this PR introduces

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review the changelog entry

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+v0.3.0:
```
Comment (IsaacHardy) [inline CHANGELOG.md]: So I thought we were abandoning the change-log?

Comment (conrad-vanl) [reply CHANGELOG.md]: adding to the changelog is still in the pull request template, so I did it 😇

Comment (IsaacHardy) [reply CHANGELOG.md]: Not if you remove it 🙈

**Comment by conrad-vanl** [comment]
@IsaacHardy coverage can start to be improved once we get enzyme pr moved.

**Pull request merged**